WBooks/Book Details/BookDetailsViewController.swift

```diff
@@ -41,7 +41,28 @@ class BookDetailsViewController: UIViewController {
     func onLoadRentsSuccess(rents: [Rent]) {
         _viewModel.onLoadRentsSuccess(rents: rents)
         _view.setAvailability(status: _viewModel.status)
-        _view.rent.addTarget(self, action: #selector(rent), for: .touchUpInside)
+        setRentButton()
+//        _view.rent.addTarget(self, action: #selector(rent), for: .touchUpInside)
```

Comment:
Delete commented code

Reply:
Why are you calling this function here and not when you the view is loaded?? What happens if the user press the button before the request is completed?

Reply:
Originally availability had no default value and thus made the button only functional when the request was complete (temporarily), then we changed and improved that, and the button can now handle that case, but forgot to move this to view did load. My bad.